finesse/gui/temp_control.py

```diff
@@ -161,6 +173,25 @@ def __init__(self, num_channels: int) -> None:
         layout = self._create_controls()
         self.setLayout(layout)
 
+        self._begin_polling()
+
+        pub.subscribe(self._begin_polling, "temperature_monitor.open")
+        pub.subscribe(self._end_polling, "temperature_monitor.close")
+        pub.subscribe(self._update_pt100s, "temperature_monitor.data.response")
+
+    def _begin_polling(self) -> None:
+        """Initiate polling the DP9800 device."""
+        self._poll_light.timer.start(2000)
```

Comment:
Could we make this 2000 a class attribute,  with the appropriate docstring, so it is easy to identify and change, if needed? Otherwise, it just pops up out of the blue with no clear meaning.

Reply:
I'd call `self._poll_light_timer.setInterval()` in the constructor and let the user pass the poll interval as an argument (with a sensible default value).